src/vmeasure.jl

```diff
@@ -0,0 +1,70 @@
+"""V-measure of contingency table"""
+function _vmeasure(A::Matrix{Int}; β = 1.0)
```

Comment:
`_vmeasure(A::AbstractMatrix{Int}, β::Real = 1.0)`
Type inference issues related to keyword arguments should be mostly solved in Julia v0.7, but here it's easy to avoid them.